README.md

```diff
@@ -1 +1,155 @@
-# mlflow
\ No newline at end of file
+# Run your personal MLFlow server on top of Neu.ro platform with neuro-flow
+
+This is a [`neuro-flow`](https://github.com/neuro-inc/neuro-flow) action launching an instance of [MLFlow tracking server](https://www.mlflow.org/docs/latest/tracking.html).
+You might use it to track your ML experiments, model trainings as well as deploying those models to production using our [MLFlow2Seldon intrgration](https://github.com/neuro-inc/mlops-k8s-mlflow2seldon)
```

Comment:
`track your ML experiments, model trainings ` → `track your ML experiments AND model trainings`
`those models` → `the models`
`intrgration` → `integration

Reply:
sure, fixed in   5469603